tests/helpers/test_delete_instance.py

```diff
@@ -35,7 +35,7 @@ def test_failed_deletion(mocker):
     cr = mocker.Mock(spec=testinfra.backend.base.CommandResult)
     mocker.patch('testinfra.host.Host.run', return_value=cr)
 
-    server = {'id': 'foo', 'name': 'myserver'}
+    server = [{'ID': 'foo', 'Name': 'myserver'}]
```

Comment:
Capital ID?

Reply:
Yes, `ID`, and `Name` are the correct keys when the openstack cli is using the `list` sub-command to get resources.
```
openstack volume list -f json 
[ {
    "Status": "deleting",
    "Size": 1,
    "Attached to": "",
    "ID": "c331df48-1e02-4fc8-bac6-1822fbd01990",
    "Name": "test_volume_compute1"
  }
]
```
The other test in this file has been corrected to reflect these keys. Thanks for catching this one.